metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFastTableService.java

```diff
@@ -51,16 +45,17 @@
  * @since 1.0.0
  */
 @Slf4j
+@Transactional(readOnly = true)
 public class HiveConnectorFastTableService extends HiveConnectorTableService {
```

Comment:
As I read through this PR an idea is forming in my head that what we probably need to do in the main application is flag the `@ComponentScan` on the `@SpringBootApplication` class to exclude packages. Then we make our connectors all under the packages to exclude. Something like `com.netflix.metacat.connector.impl.{hive,redshift,ect}`. That way these are isolated.